Dockerfile

```diff
@@ -27,6 +27,7 @@ FROM docker.io/library/golang:1.23.0-alpine3.20
 RUN apk add --no-cache \
     runuser \
     git \
+    gcompat \
```

Comment:
Why this addition?

Reply:
Ok, catching up with all comments. Thanks for identifying this Hugo requirement 🙌🏻.